config/settings/test.yml

```diff
@@ -3,3 +3,4 @@ govuk_notify:
   welcome_email_template_id: please_change_me
 system_authentication_token: "Ge32"
 gcp_api_key: please_change_me
+test_setup: false
```

Comment:
I don't really feel like this is an app concern ... tests should only ever be run in the `testing` Rails env, for example. I think a bespoke env var would be more appropriate, no?

Reply:
👍